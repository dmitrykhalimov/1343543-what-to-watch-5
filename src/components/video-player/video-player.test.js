import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

describe(`VideoPlayer`, () => {
  it(`Should VideoPlayer render correctly, isPlaying`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            src={``}
            preview={``}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
