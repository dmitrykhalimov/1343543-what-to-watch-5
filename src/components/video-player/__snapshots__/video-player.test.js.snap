// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should VideoPlayer render correctly, isPlaying 1`] = `
<div
  className="small-movie-card__image"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video
    autoPlay={false}
    height="175"
    muted={true}
    poster=""
    src=""
    width="280"
  />
</div>
`;
